scripts/googleAppScripts/SheetToGithubIssue.gs

```diff
@@ -6,38 +6,62 @@ function onFormSubmit(e) {
-                    "\n\n### Facebook Link\n" + (facebookLink || 'N/A');
-
-    // Use the initiative name as the issue title
-    var issueTitle = initiativeName;
+  // todo: adjust the form and add these fields
+  var initiativeDetails = '_No response_';
+  var validationLink = '_No response_';
+  var websiteLink = '_No response_';
+  var phoneNumber = '_No response_';
+  var email = '_No response_';
+  var formLink = formResponse['Google Form link'][0];
+  var documentLink = formResponse['Google Doc link'][0];
+  var linkedinLink = '_No response_';
+  var youtubeLink = '_No response_';
+  var donationLink = '_No response_';
+  var logoLink = '_No response_';
+  var tags = '_No response_';
+
+  // Function to format the value or return "_No response_"
+  function formatValue(value) {
+    return value.trim() ? value.trim() : "_No response_";
+  }
+
+  // Construct the issue body using the form response data with markdown
+  var issueBody = `### Initiative Name\n${formatValue(initiativeName)}\n\n` +
+                  `### Initiative Description\n${formatValue(description)}\n\n` +
+                  `### Initiative Details\n${formatValue(initiativeDetails)}\n\n` +
+                  `### Initiative Validation Details\n${formatValue(validationLink)}\n\n` +
+                  `### Initiative Category\n${formatValue(category)}\n\n` +
+                  `### Initiative main URL\n${formatValue(mainLink)}\n\n` +
+                  `### Initiative Website\n${formatValue(websiteLink)}\n\n` +
+                  `### Initiative Phone Number\n${formatValue(phoneNumber)}\n\n` +
+                  `### Initiative E-Mail\n${formatValue(email)}\n\n` +
+                  `### Initiative WhatsApp\n${formatValue(whatsappLink)}\n\n` +
+                  `### Initiative Telegram\n${formatValue(telegramLink)}\n\n` +
+                  `### Initiative Drive\n${formatValue(driveLink)}\n\n` +
+                  `### Initiative Form\n${formatValue(formLink)}\n\n` +
+                  `### Initiative Document\n${formatValue(documentLink)}\n\n` +
+                  `### Initiative Discord\n${formatValue(discordLink)}\n\n` +
+                  `### Initiative Facebook\n${formatValue(facebookLink)}\n\n` +
+                  `### Initiative Instagram\n${formatValue(instagramLink)}\n\n` +
+                  `### Initiative TikTok\n${formatValue(tiktokLink)}\n\n` +
+                  `### Initiative X (Twitter)\n${formatValue(twitterLink)}\n\n` +
+                  `### Initiative LinkedIn\n${formatValue(linkedinLink)}\n\n` +
+                  `### Initiative Youtube Page / Channel\n${formatValue(youtubeLink)}\n\n` +
+                  `### Initiative Donation Link\n${formatValue(donationLink)}\n\n` +
+                  `### Initiative logo\n${formatValue(logoLink)}\n\n` +
+                  `### Initiative Tags\n${formatValue(tags)}`;
+
+    // Use the initiative name as the issue title + the required prefix
```

Comment:
Ideally we'd have an additional GForms field for the display name in Hebrew - @4tal can you add Daniel and me as admins / editors on the form?

Reply:
@DanielShterenberg could you add such a field to the Google form? We currently have full fidelity with the GitHub form body, but we're missing the GitHub form title (which we'd map to this extra gForm field).